web-ui/src/main/resources/catalog/templates/editor/accessManager.html

```diff
@@ -0,0 +1,99 @@
+<div class="container" id="gn-access-manager-container">
+  <br/>
+  <br/>
+  <br/>
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span data-translate="">accessManagerTitle</span>
+        </div>
+
+        <div class="panel-body" data-ng-controller="GnAccessManagerController">
+
+          <div class="row">
+            <div class="col-md-6">
+              <label class="control-label"
+                     data-translate>searchByTitle</label>
+              <input type="text"
```

Comment:
This is definitely lacking a debounce, considering the browser freezes up when receiving new records.

Reply:
Added